contracts/GraphTokenLock.sol

```diff
@@ -371,6 +372,7 @@ abstract contract GraphTokenLock is Ownable, IGraphTokenLock {
         uint256 unvestedAmount = managedAmount.sub(vestedAmount());
         require(unvestedAmount > 0, "No available unvested amount");
 
+        revokedAmount = revokedAmount.add(unvestedAmount);
```

Comment:
Since we know that revoke can only be called once, we could just do:

```
revokedAmount = unvestedAmount;
```

however, the way you did it might be easier to read. So I am good either way